doc/examples/markup.mli

```diff
@@ -52,46 +52,43 @@ module type Foo = sig
   val bar : string
   (** This comment is associated to bar. *)
 
-  class cl :
-    object
-
-      (** Interesting information about cl *)
-    end
+  class cl : object
+    
+(** Interesting information about cl *)
```

Comment:
This seems an unfortunate change.

Reply:
Good catch!

Reply:
Fixed in https://github.com/ocaml-ppx/ocamlformat/pull/2387